win32ss/user/user32/windows/cursoricon.c

```diff
@@ -1148,6 +1163,21 @@ BITMAP_LoadImageW(
         goto end;
     CopyMemory(pbmiCopy, pbmi, iBMISize);
 
+    TRACE("Size Image %d, Size Header %d, ResSize %d\n",
+        pbmiCopy->bmiHeader.biSizeImage, pbmiCopy->bmiHeader.biSize, ResSize);
+
+    /* Test if this is a GCC windres.exe compiled 16 or 32 bpp bitmap using
```

Comment:
I guess we should add a `HACK: ` or something in this comment, in order to make it obvious that this adjustment is to workaround a binutils bug.
(Technically, windres.exe is part of binutils, not GCC which is just the C/C++ compiler itself.)

Reply:
Now revised to say "binutils" instead of "GCC" and added "HACK:". Thanks.